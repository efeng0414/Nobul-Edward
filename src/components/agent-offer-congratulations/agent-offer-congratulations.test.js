import React from "react";
import renderer from "react-test-renderer";
import AgentOfferCongratulations from "./";

describe("Agent Offer Congratulations Component", () => {
  const componentProps = {
    intl: global.getIntl(),
    onNext: () => {}
  };
  const componentJSX = <AgentOfferCongratulations {...componentProps} />;

  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });

  // REMOVE_PROMOTED remove test.
  // describe("button click event", () => {
  //   const componentMount = global.Enzyme.mount(componentJSX);
  //   it("should trigger onNext function when button is clicked", () => {
  //     const mockFn = jest.fn();
  //     componentMount.setProps({ onNext: mockFn });
  //     componentMount
  //       .find("Button")
  //       .props()
  //       .onClick();
  //     expect(mockFn).toHaveBeenCalled();
  //   });
  // });
});
