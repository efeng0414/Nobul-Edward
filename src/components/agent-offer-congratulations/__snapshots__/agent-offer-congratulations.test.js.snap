// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agent Offer Congratulations Component Component Snapshot should match existing snapshot 1`] = `
<div
  className="proposal-congratulations"
>
  <img
    alt="CONGRATULATIONS"
    src="file-mock"
  />
  <p
    className="proposal-congratulations-title"
  >
    CONGRATULATIONS
  </p>
  <p
    className="proposal-congratulations-subtitle"
  >
    Your proposal is on it's way!
  </p>
</div>
`;
